corehq/apps/ota/tests/test_utils.py

```diff
@@ -292,23 +274,15 @@ def test_get_restore_user_as_super_user(self):
             get_restore_user(
                 self.domain,
                 self.web_user,
-                self.super_user.username,
+                self.super_user
             ),
             OTARestoreWebUser,
         )
 
-    def test_get_restore_user_not_found(self):
-        with self.assertRaises(Exception):
-            get_restore_user(
-                self.domain,
-                self.web_user,
-                '{}@wrong-domain'.format(self.commcare_user.raw_username, self.domain)
-            )
-
```

Comment:
how come we're getting rid of this test and the other two?

Reply:
They were testing behavior related to passing in bad usernames to `get_restore_user`, which is no longer possible, since they now accept user objects.  There's a separate test which tests that the user is associated with the correct domain.  Part of that logic was moved [here](https://github.com/dimagi/commcare-hq/pull/18131/files#diff-291dfd3802bf0daa51e598788d5ca13bR179) and that is now untested.  (It's pretty straightforward though, so I'm not concerned about that).